### PR TITLE
Updated misconfigured dispatch

### PR DESCRIPTION
**Pull request by MatthewChon**
#### Description of changes

Change verifytotptoken() to emit an 'verify' event instead of 'signIn'.
Added the case 'verify' to the listener.

#### Issue #, if available

https://github.com/aws-amplify/amplify-js/issues/9750

#### Description of how you validated changes
i reproduced the issue and from there, I reran the application with the modified changes.
The hub now console logs 'verify' instead of 'signIn'.

#### Checklist

- [X] PR description included
- [X] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by AllanZhengYP — COMMENTED** [review]
This change looks good to me. But I'm not sure whether stop emitting `signIn` is a breaking change for those expecting this event. I'd defer to other approvers.

**Comment by MatthewChon** [comment]
Ok, that sounds good!

**Review by elorzafe — APPROVED** [review]
LGTM thanks @MatthewChon 🎖️

**Review by AllanZhengYP — DISMISSED** [review]

File: packages/auth/src/Auth.ts [hunk 1]
```diff
@@ -1138,10 +1139,11 @@ export class AuthClass {
 					return;
 				},
 				onSuccess: data => {
+					dispatchAuthEvent('signIn', user, ``);
```
Comment (haverchuck) [inline packages/auth/src/Auth.ts:1142]: @MatthewChon Thanks for this PR. Why would we not continue to log the message `A user ${user.getUsername()} has been signed in` when emitting the signIn event?

**Review by haverchuck — CHANGES_REQUESTED** [review]
@MatthewChon Please see inline [comment](https://github.com/aws-amplify/amplify-js/pull/10511/files#r1049975351).

File: packages/auth/src/Auth.ts (hunk 1 above)
Comment (haverchuck) [inline packages/auth/src/Auth.ts:1142]:
```suggestion
					dispatchAuthEvent('signIn', user, `A user ${user.getUsername()} has been signed in`);
```

**Review by haverchuck — APPROVED** [review]

**Review by elorzafe — APPROVED** [review]

**Comment by cnorthfield** [comment]
Could really do with the signin event _not_ being emitted 🙈

**Pull request merged**